src/index.html

```diff
@@ -87,7 +87,7 @@ <h3 class="panel-title">Category</h3>
 <!-- Libraries -->
 <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
 
-<script src='https://api.mapbox.com/mapbox-gl-js/v2.0.0/mapbox-gl.js'></script>
```

Comment:
I think we could add this back in here at this point, but I think I prefer keeping as many dependencies as possible in `package.json`.